docs/sources/flow/getting-started/collect-opentelemetry-data.md

```diff
@@ -144,3 +150,182 @@ otelcol.receiver.otlp "example" {
+usually sent to one or more _processor components_ that perform various
+transformations on the data.
+
+Ensuring data is batched is a production-readiness step to improve the
+compression of data and reduce the number of outgoing network requests to
+external systems.
+
+In this task, we will configure an [otelcol.processor.batch][] component to
+batch data before sending it to our exporter.
+
+> Refer to the list of available [Components][] for the full list of
+> `otelcol.processor` components that can be used to process OpenTelemetry
+> data. You can chain processors by having one processor send data to another
+> processor.
+>
+> [Components]: {{< relref "../reference/components/" >}}
+
+To configure an `otelcol.processor.batch` component, complete the following
+steps:
+
+1. Follow [Configure an OpenTelemetry
+   exporter](#configure-an-opentelemetry-exporter) to ensure received data can
+   be written to an external system.
+
+2. Add the following `otelcol.processor.batch` component into your
+   configuration file:
+
+   ```river
+   otelcol.processor.batch "PROCESSOR_LABEL" {
+     output {
+       metrics = [otelcol.exporter.otlp.EXPORTER_LABEL.input]
+       logs    = [otelcol.exporter.otlp.EXPORTER_LABEL.input]
+       traces  = [otelcol.exporter.otlp.EXPORTER_LABEL.input]
+     }
+   }
+   ```
+
+    1. Replace `PROCESSOR_LABEL` with a label to use for the component, such as
+       `default`. The label chosen must be unique across all
+       `otelcol.processor.batch` components in the same configuration file.
+
+    2. Replace `EXPORTER_LABEL` with the label for your existing
+       `otelcol.exporter.otlp` component.
+
+    3. To disable one of the telemetry types, set the relevant type in the
+       `output` block to the empty list, such as `metrics = []`.
+
+    4. To send batched data to another processor, replace the components in the
+       `output` list with the processor components to use.
+
+The following example demonstrates configuring a sequence of
+`otelcol.processor` components before ultimately being exported:
+
+```river
+otelcol.processor.memory_limiter "default" {
+  check_interval = "1s"
+  limit          = "1GiB"
+
+  output {
+    metrics = [otelcol.processor.batch.default.input]
+    logs    = [otelcol.processor.batch.default.input]
+    traces  = [otelcol.processor.batch.default.input]
+  }
+}
+
+otelcol.processor.batch "default" {
+  output {
+    metrics = [otelcol.exporter.otlp.default.input]
+    logs    = [otelcol.exporter.otlp.default.input]
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "my-otlp-grpc-server:4317"
+  }
+}
+```
+
+For more information on configuring OpenTelemetry data batching, refer to
+[otelcol.processor.batch][].
+
+## Configure an OpenTelemetry receiver
+
+Grafana Agent Flow can be configured to receive OpenTelemetry metrics, logs,
```

Comment:
Same comment as the other PR: OTLP should be used when covering the data protocol.